linkis-commons/linkis-storage/src/main/java/org/apache/linkis/storage/domain/MethodEntity.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.linkis.storage.domain;
+
+/**
+ * Engine unique Id(engine唯一的Id)
+ *
+ * <p>Fs type(fs类型)
+ *
+ * <p>Create a user to start the corresponding jvm user(创建用户为对应启动的jvm用户)
+ *
+ * <p>Proxy user(代理用户)
+ *
+ * <p>client Ip for whitelist control(ip用于白名单控制)
+ *
+ * <p>Method name called(调用的方法名)
+ *
+ * <p>Method parameter(方法参数)
+ */
+public class MethodEntity {
+  public long id;
+  public String fsType;
+  public String creatorUser;
+  public String proxyUser;
+  private String clientIp;
+  public String methodName;
+  public Object[] params;
```

Comment:
the field should be private?

Reply:
yes, I have finished modify field variable to private , Thanks. Sorry for later reply .

Reply:
I have resolved the conflict with dev-1.4.0.